sdcm/nemesis.py

```diff
@@ -180,6 +180,30 @@ class DefaultValue:  # pylint: disable=too-few-public-methods
     ...
 
 
+def use_data_nodes_as_target(func):
```

Comment:
consider naming in the form of: `target_data_nodes`, `target_zero_nodes` and `target_all_nodes`

Reply:
done